### PR TITLE
Update mvn pom and other little updates;

### PR DESCRIPTION
**Pull request by RSheremeta**
- Added dependencides for **Allure, Log4j, JUnit5** (this was requiring one more);
- Added chromedriver file to project dir;
- **BaseConfiguration.class** - Changed path to chromedriver to more flexible.

### CODE REVIEW

File: allure-results/4a0b789f-533e-4fc1-8d3d-4c1509ff2403-container.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"uuid":"4a0b789f-533e-4fc1-8d3d-4c1509ff2403","name":"searchForGoogle()","children":["6df04e68-6c41-4126-895e-b5e424c0617c"],"befores":[],"afters":[],"links":[],"start":1586255471611,"stop":1586255477403}
```
Comment (wizzardio-man) [inline allure-results/4a0b789f-533e-4fc1-8d3d-4c1509ff2403-container.json:1]: Do you need this *.json file here?

File: allure-results/5a8e0f99-9e78-40c0-b901-df3b42449f04-container.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"uuid":"5a8e0f99-9e78-40c0-b901-df3b42449f04","name":"GoogleTest","children":["6df04e68-6c41-4126-895e-b5e424c0617c"],"befores":[],"afters":[],"links":[],"start":1586255466221,"stop":1586255477424}
```
Comment (wizzardio-man) [inline allure-results/5a8e0f99-9e78-40c0-b901-df3b42449f04-container.json:1]: Same for this one?

File: allure-results/6df04e68-6c41-4126-895e-b5e424c0617c-result.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"name":"searchForGoogle()","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1586255471653,"stop":1586255477241,"uuid":"6df04e68-6c41-4126-895e-b5e424c0617c","historyId":"48b5139eb06e2859c082eb21e81d6e72","fullName":"google.GoogleTest.searchForGoogle","labels":[{"name":"host","value":"MacBook-Pro-Roman.local"},{"name":"thread","value":"1947@MacBook-Pro-Roman.local.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"google.GoogleTest"},{"name":"testClass","value":"google.GoogleTest"},{"name":"testMethod","value":"searchForGoogle"},{"name":"suite","value":"google.GoogleTest"}],"links":[]}
```
Comment (wizzardio-man) [inline allure-results/6df04e68-6c41-4126-895e-b5e424c0617c-result.json:1]: And for this one?

File: src/main/java/base/configuration/BaseConfiguration.java [hunk 4]
```diff
@@ -16,7 +16,8 @@
     @Bean
     public WebDriver webDriver() throws MalformedURLException, IOException {
         // TODO: create ENV variable for this one
-        System.setProperty("webdriver.chrome.driver", "C:\\Users\\chromedriver.exe");
+        System.setProperty("webdriver.chrome.driver",
```
Comment (wizzardio-man) [inline src/main/java/base/configuration/BaseConfiguration.java:19]: It looks like you modified the path for macOS System. But unfortunatly I'm using Windows OS. So this step will fail for me. 

image

File: src/main/java/base/configuration/BaseConfiguration.java [hunk 5]
```diff
@@ -16,7 +16,8 @@
...
+          System.getProperty("user.dir") + "//src//main//java//base//configuration//chromedriver");
```
Comment (wizzardio-man) [inline src/main/java/base/configuration/BaseConfiguration.java:20]: Regarding the `chromedriver` that you included. Can you put it into the `resources` folder?
 It's not the best approach to mix binary files and *.java classes in one folder.

**Comment by RSheremeta** [comment]
@wizzardio-man resolved all your objections and pushed the commit.

**Pull request merged**